app/Renderers/Layouts/Map/Marker.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Renderers\Layouts\Map;
+
+use App\Renderers\Layouts\Columns\Standard;
+use App\Renderers\Layouts\Layout;
+
+class Marker extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'name' => [
+                'key' => 'name',
+                'label' => 'crud.fields.name',
+                'render' => function ($model) {
+                    return $model->tooltipedLink($model->markerTitle());
+                },
+            ],
+            'entity_id' => [
+                'label' => 'crud.fields.entity',
+                'render' => function ($model) {
+                    if ($model->entity_id) {
+                        return $model->tooltipedLink($model->entity->name);
```

Comment:
Missing a check that entity isn't empty, otherwise for a non-admin this page will crash if entity_id is set, but the user doesn't have permission to see the marker.

Can be simplified by
`return $model->entity?->tooltipedLink()`, that way the tooltips also work